lib/gap_intelligence/client/requestable.rb

```diff
@@ -8,21 +8,21 @@ module Requestable
     # @yield [req] The Faraday request
     def perform_request(method, path, options = {}, &block)
       record_class = options.delete(:record_class)
+      response_processor = options.delete(:response_processor)
```

Comment:
<img width="949" alt="gapintelligence-gem_requestable_rb_at_ad1a36a9c978c5021bd2f933942c3e1ec31c0189_ _gapintelligence_gapintelligence-gem" src="https://cloud.githubusercontent.com/assets/292051/16161619/f459fcc4-34e7-11e6-9666-697571c9f85d.png">

@shavkat-samatov could you please explain how this variable is being used to achieve you described above?